virttest/utils_misc.py

```diff
@@ -3474,6 +3474,30 @@ def check_device_driver(pci_id, driver_type):
     return driver == driver_type
 
 
+def get_guest_os_datetime(session, os_type):
```

Comment:
easy way to get seconds since epoch:

python -c "import datetime; print (datetime.datetime(2012,04,01,0,0) - datetime.datetime(1970,1,1)).total_seconds()"